package aa.exception.handlingfinnaly_rules;

public class Test02 {

}
